content/docs/guides/vote_poll_sdk/dapp_creation/_index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Poll/Vote dapp creation
+date: 2022-01-03T11:00:00+1100
+weight: 1
+---
+
+# How to Use the WakuConnect Poll/Vote SDK
+
+This part is the same for both Poll and Vote SDK.
+
+To demonstrate how to use the WakuConnect Poll/Vote SDK in your dApp, we will create a TypeScript React app from scratch.
+
+You can then adapt the steps depending on your dApp configuration and build setup.
+
+Only token holders can create & answer polls.
```

Comment:
This guide is for both vote and poll SDK, rights? So maybe change to:
`The poll and vote SDK features can only be used by token holders, you must pass the ERC20 token contract of your choice when using the SDK.`